### PR TITLE
Add Enhance Image documentation

### PR DESCRIPTION
**Pull request by colepeters**
Not to be merged until Enhance Image 1.0.0 stable is released.

### CODE REVIEW

**Comment by colepeters** [comment]
Holding this until aws-lite is integrated into arc-image-plugin (https://github.com/enhance-dev/arc-image-plugin/pull/9) to prevent branching behaviour based on Node version

File: app/docs/md/enhance-ui/image.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+></enhance-image>
+```
+
+- At viewports at least 96em wide, the generated image closest to `25vw` in width will be used; at viewports between 48–96em wide, the generated image closest to `50vw` in width will be used; at viewports narrower than 48em, the image closest to the full width of the current viewport will be used
+
+## Usage notes
+
+### `sizes` attribute
+
+The `sizes` attribute used by Enhance Image is simply a proxy for [`HTMLImageElement.sizes`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes). This `sizes` attribute, as specified by [the HTML living standard](https://html.spec.whatwg.org/multipage/embedded-content.html#dom-img-sizes), can be tough to wrap your head around in certain cases. However, we believe it’s beneficial to remain close to the web platform, as it avoids the need to learn brittle abstractions and thus keeps knowledge reusable. As such, when you learn how to work with Enhance Image’s `sizes` attribute, you’re actually learning how to work with the Image element’s `sizes` attribute — warts and all.
```
Comment (kristoferjoseph) [inline app/docs/md/enhance-ui/image.md:210]: Love this breakdown. Well put!

Comment (colepeters) [reply app/docs/md/enhance-ui/image.md:210]: Thanks so much! It took quiiiite a while to walk this tightrope ;)

**Review by kristoferjoseph — APPROVED** [review]
This has taken lots of consideration and research to get to this point and it is really nice to see how this complete solution has come together.

**Pull request merged**